Automattic/wp-calypso
# Verbum Comments: Add new verbum related settings

Related to #

## Proposed Changes

* These changes aim to add four new settings for simple sites that will allow users to customize their Verbum Comments experience:

1. `Let visitors use a WordPress.com or Facebook account to comment.` - toggles Verbum Comments ( switches to Wordpress core comment experience)
2. `Enable blocks in comments.` - toggles Gutenberg blocks in comments.
3. `Comment form introduction` - customizes comment header text (e.g. Leave a comment)
4. `Color Scheme` - customizes the color scheme of Verbum comments

<img width="725" alt="CleanShot 2024-03-21 at 12 45 40@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/40e42240-b3aa-4460-843b-13c90dc4468a">


## Testing Instructions



* Use live link or checkout this branch locally
* Build and sync this Jetpack PR to your sandbox: https://github.com/Automattic/jetpack/pull/36505
* Sandbox your simple site and public api.
* Navigate to `settings/discussion/{SITE_SLUG}` and change the new settings.
* Verify the changes are reflects in the comments section for site.
* Test `{SITE_SLUG}/wp-admin/options-discussion.php` is make sure the `Greeting Text` input field is still working correctly.

## Pre-merge Checklist
